src/components/ec-letter-icon/ec-letter-icon.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <component
+    :is="isClickable ? 'button' : 'div'"
+    data-test="ec-letter-icon"
+    class="ec-letter-icon"
+    :class="{ 'ec-letter-icon--clickable': isClickable }"
+    :style="containerSize"
+  >
+    <span
+      data-test="ec-letter-icon__text"
+      class="ec-letter-icon__text"
+      :style="letterSize"
+    >
+      {{ firstLetter }}
+    </span>
+  </component>
+</template>
+
+<script setup lang="ts">
+import { computed, type StyleValue } from 'vue';
+
+import type { LetterIconProps } from './types';
+
+const props = withDefaults(defineProps<LetterIconProps>(), {
+  size: 24,
+  isClickable: false,
+});
+
+const firstLetter = computed(() => (props.text[0] ? props.text[0].toUpperCase() : ''));
+
+// Design request: font-size has to be always half of circle size plus 2 px
+const fontSize = computed(() => props.size / 2 + 2);
```

Comment:
In this case we need to check if `props.size` is even, otherwise throw an error. I'd better introduce sizes as we have for button for example. Could you please clarify with Alba if the set of sizes is defined from design side?

Reply:
Right now, we will have 2 sizes for this component:
- 24px in sidebar
- 32px in beneficiaries details

so yes, maybe we can have an enum for it.

but just out of curiosity: why should we throw an error if size is not even? Font size can have decimals

Reply:
I believe we can't have odd sizes in our design, everything is even except border radius.
On top of that we never know how particular browser consider [decimals]px.

Reply:
okay! I'm going to add an enum then